playbooks/run-openstack-ansible-security.yml

```diff
@@ -0,0 +1,13 @@
+---
```

Comment:
Why do we need a playbook and not tie this up with main ursula run in site.yml ? don;t we want to run this and enforce the os hardening on every ursula run?